tests/unit/components/Navigator/QuickNavigationModal.spec.js

```diff
@@ -9,37 +9,89 @@
 */
 
 import { shallowMount } from '@vue/test-utils';
+import NavigatorLeafIcon from '@/components/Navigator/NavigatorLeafIcon.vue';
 import QuickNavigationHighlighter from '@/components/Navigator/QuickNavigationHighlighter.vue';
 import QuickNavigationModal from '@/components/Navigator/QuickNavigationModal.vue';
 import Reference from '@/components/ContentNode/Reference.vue';
 
+jest.mock('docc-render/utils/theme-settings', () => ({
+  getSetting: () => true,
+}));
```

Comment:
This fallback may be more useful than just returning true :)

```suggestion
jest.mock('docc-render/utils/theme-settings', () => ({
  getSetting: jest.fn((_, fallback) => fallback),
}));
```

Reply:
https://github.com/apple/swift-docc-render/pull/396/commits/aaa7e4f770e7fbb1c3daeb07fb0ea43867bc231f